metadata-models/src/main/pegasus/com/linkedin/datajob/DataFlowInfo.pdl

```diff
@@ -38,4 +39,15 @@ record DataFlowInfo includes CustomProperties, ExternalReference {
     "queryByDefault": false
   }
   project: optional string
-}
\ No newline at end of file
+  
+  /**
+   * Optional list of jobs contained in this data flow
+   */
+  @Relationship = {
+    "/*": {
+      "name": "Contains",
+      "entityTypes": [ "dataJob" ]
+    }
+  }
+  jobs: optional array[DataJobUrn]
```

Comment:
There is this one already https://github.com/datahub-project/datahub/blob/c5f1d2c9bdf92684ebec19cf972b231e832b126a/metadata-models/src/main/pegasus/com/linkedin/datajob/DataJobInfo.pdl#L44

keeping references on both sides of the relationship could be hard to maintain, couldn't it?